name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/PortfolioPerformanceSearchProvider.java

```diff
@@ -84,7 +87,7 @@ public static Optional<ResultItem> from(JSONObject json)
                 answer.markets.sort(new ByExchangeComparator());
             }
 
-            return Optional.of(answer.markets.size() == 1 ? answer.markets.get(0) : answer);
+            return Optional.of(answer.markets.size() > 0 ? answer.markets.get(0) : answer);
```

Comment:
The idea here was: if the instrument is traded on one market only, then simplify the UI (= do not show the instrument, then then a list of one market --> instead just show the one market directly). However, if there are multiple markets, then of course we need to return the full list.